float32 X_left
float32 Y_left
bool status
float32 X_right
float32 Y_right
